docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,282 @@
+     f"but fold_multiplier=3:  \n {scale_factors_diff_fold_multiplier[-2]}")
+```
+
+
+Both the number of noise scaled circuits and scale factor vectors are changed when a different value for `degree` is used while keeping everything else the same.
+
+
+```{code-cell} ipython3
+
+
+scale_factors_diff_degree = get_scale_factor_vectors(
+   circuit,
+   degree=3,
+   fold_multiplier=2)
+
+print(f"Total number of noise scaled circuits created: "
+      f"{len(scale_factors_diff_degree)}")
+```
+
+
+Thus, even though `degree` and `fold_multiplier` are required to use {func}`.execute_with_lre`, they function as a tunable
+hyperparameter affecting the performance of the technique.
+
+
+## Chunking a circuit into fewer layers
+
+
+When you have a large circuit, the size of the sample matrix increases as the number of monomial terms scale polynomially. The size of the sample matrix influences sampling cost. In such a case, a circuit of 100 layers could be grouped into 4 chunks where each chunk consists of multiple collated layers. The noise scaling function {func}`.multivariate_layer_scaling`
+treats each chunk as a layer to be scaled when the parameter `num_chunks` is used. Thus, for the 100 layer circuit grouped into 4 chunks with `degree = 2` and `fold_multiplier = 2`, only 15 noise-scaled circuits are created i.e. sample matrix is reduced to dimension $15 \times 15$.
+
+
+```{caution}
+Reducing the sampling cost by chunking the circuit can affect the performance of the technique.
+```
+
+
+Suppose we want to chunk our example circuit into 2 chunks while using `degree = 2` and
+`fold_multiplier = 2`. The sample matrix defined by the monomial terms is reduced in size as chunking the circuit
+will create a new monomial basis for the extrapolating polynomial.
+
+
+$$\{1, λ_1, λ_2, λ_1^2, λ_1 λ_2, λ_2^2\}$$
+
+
+The scale factor vectors change as shown below:
+
+
+```{code-cell} ipython3
+
+scale_factors_with_chunking = get_scale_factor_vectors(
+   circuit,
+   degree=2,
+   fold_multiplier=2,
+   num_chunks=2)
+
+scale_factors_with_chunking
+```
+
+
+Thus, the total number of noise-scaled circuits is reduced by chunking the circuit into fewer layers to be folded.
+
+
+```{code-cell} ipython3
+
+print(f"Total number of noise scaled circuits with chunking: "
+      f"{len(scale_factors_with_chunking)}")
+
+
+print(f"Total number of noise scaled circuits without chunking: "
+      f"{len(scale_factors)}")
+
+
+```
+
+
+How the noise-scaled circuits are chunked differs greatly as each chunk in the circuit is now equivalent to a layer to be folded via unitary folding.
+
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import multivariate_layer_scaling
+
+# apply chunking
+chunked_circ = multivariate_layer_scaling(
+ circuit,
+ degree = 2,
+ fold_multiplier = 2,
+ num_chunks = 2)[1]
+
+
+# skip chunking
+non_chunked_circ = multivariate_layer_scaling(
+ circuit,
+ degree = 2,
+ fold_multiplier = 2)[1]
```

Comment:
This is comparing the $\lambda_1$ term for each method of scaling, right? Might be worth spelling that out.